index.html

```diff
@@ -190,14 +193,14 @@ <h2 class = "section__header">
                                     <div class = "location-in-focus__text-element__header">
                                         Featured Neighborhood: <br> Kyoto’s Arashiyama
                                     </div>
-                                    <div class = "location-in-focus__text-element__info">
+                                    <span class = "location-in-focus__text-element__info">
                                         Jump off balcony, onto stranger's head. Chase ball of string
                                         hide when guests come over. Being gorgeous with belly side
                                         up i could pee on this if i had the energy but under the bed,
                                         for attack the child, open the door, 
-                                    </div>
-                                    <a href="">View Neighborhood &#160 &#160 &#160
-                                        <img src="images/arrow.svg" alt="arrow">
+                                    </span>
+                                    <a class="blue-link-with-arrow" href="">
```

Comment:
<img width="328" alt="image" src="https://user-images.githubusercontent.com/52242019/148558541-58e5ac10-61bb-4156-8897-6575021aa878.png">
Could you pls adapt it for mobile?